kotest-assertions/kotest-assertions-json/src/commonTest/kotlin/io.kotest.assertions.json/schema/ArraySchemaTest.kt

```diff
@@ -3,11 +3,10 @@ package io.kotest.assertions.json.schema
 import io.kotest.assertions.shouldFail
 import io.kotest.core.spec.style.FunSpec
 import io.kotest.matchers.shouldBe
-import org.intellij.lang.annotations.Language
 
 class ArraySchemaTest : FunSpec(
    {
-      fun json(@Language("JSON") raw: String) = raw
+      fun json(raw: String) = raw
```

Comment:
This method and it's usage should be completely removed. It's only point was to provide the language injection 😄

Reply:
Good catch - I've fixed this in https://github.com/kotest/kotest/pull/3021/commits/c1dbc548621d9f2dc7cfac13fece6275fd08ae0d.